regexing/src/main/java/Validation.java

```diff
@@ -77,6 +94,8 @@ public boolean email(String input){
      * @return
      */
     public boolean address(String input){
-        return true;
+        Pattern p = Pattern.compile("\\d+(?:\\s[A-Z][a-z]+\\.?)+\\s?\\d*[\\n,]?(?:\\s?[A-Z][a-z]+)+,\\s[A-Z]{2}\\s\\d{5}(?:-\\d{4})?");
```

Comment:
Nicely done.  One thing with regex and text matching is that you generally should assume your data will be dirty.  So don't assume that the first letter will be upper case.  Additionally, this gets strange with DR vs Drive vs Dr.  In general just ignore case and go for the pattern.